persistent-sqlite/Database/Persist/Sqlite.hs

```diff
@@ -539,7 +541,7 @@ getCopyTable :: [EntityDef]
              -> EntityDef
              -> IO [(Bool, Text)]
 getCopyTable allDefs getter def = do
```

Comment:
The changes to this function were non-trivial. A naive change to this function would allow tables in different databases to clobber each other when copying them into the temporary schema. So we do some manual namespacing by prefixing the table name with `<schema>_` when copying it to the temporary schema.